src/main/java/com/jcabi/http/mock/MkAnswer.java

```diff
@@ -209,6 +220,18 @@ public String toString() {
                 body.getBytes(MkAnswer.Simple.CHARSET)
             );
         }
+        /**
+         * Make a copy of this answer, with another body.
+         * @param body Body
+         * @return New answer
+         */
+        public MkAnswer.Simple withBody(final byte[] body) {
+            return new MkAnswer.Simple(
+                    this.code,
+                    this.hdrs,
+                    body
```

Comment:
@pstorch according to http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#indentation these lines should be joined